### ISSUE TITLE
log-cache-expvar-forwarder timestamp format neither UNIX nor IS08601

### ISSUE THREAD

**Issue by jomora** [cloudfoundry/log-cache-release#20]
Hi,

the `log-cache-expvar-forwarder` creates syslog messages in the following format:
```
{"timestamp":1542276581325597486,"name":"last-capi-v3-apps-latency","value":0.000000,"source_id":"log-cache-cf-auth-proxy","type":"gauge"}
```

It seems to me that the timestamp is broken due to a missing dot.
I would expect the timestamp to have the UNIX Epoch time format, i.e.  `1542276581.325597486`.
Running `date -d @1542276581.325597486` indicates that this was probably intended.

What do you think?

Cheers
Jonas

**Comment by toddboom** [comment]
@jomora thanks for submitting this issue! it looks like we're just outputting `timestamp` in nanoseconds as an int64. would it be preferable to output this in RFC3339?

/cc @jtuchscherer

**Comment by jtuchscherer** [comment]
@toddboom is this still a problem now that the expvar forwarder is sending its messages to the local loggregator agent?

**Comment by toddboom** [comment]
@jtuchscherer great question. we should check to confirm, but i think you may be correct.

**Comment by jomora** [comment]
@toddboom, @jtuchscherer  as far as I understand, according to the [format](https://github.com/cloudfoundry/syslog-release#format) the message sent to the loggregator agent is appended as `$MESSAGE`.
My goal is to parse the timestamp within this message. If the timestamp does not have a standardized format it's hard to parse it.
In the end, it doesn't matter to me  whether it's a UNIX timestamp or adheres to RFC3339 as long as it's standardized. 
Currently, a typical syslog message looks like:
```
<14>1 2018-11-20T13:10:54.592657Z 10.0.65.104 log-cache-expvar-forwarder rs2 - [instance@47450 director="bosh-director" deployment="cf" group="doppler" az="z1" id="18fb44b5-9ac8-4913-a2e1-3155503003a8"] {"timestamp":1542719454377019733,"name":"last-capi-v2-list-service-instances-latency","value":0.000000,"source_id":"log-cache-cf-auth-proxy","type":"gauge"}
--
```

**Comment by jtuchscherer** [comment]
We will remove the expvar forwarder in favor of a prometheus exposition endpoint and a prom scraper. We will make sure that the timestamp is properly formatted in that new implementation. Closing this for now. I created this story to track the expvar->prom-scraper work: https://www.pivotaltracker.com/story/show/162397765
